apps/io-services-cms-webapp/src/main.ts

```diff
@@ -11,3 +18,15 @@ export const httpEntryPoint = pipe(
   createWebServer,
   expressToAzureFunction
 );
+
+const config = getConfigOrThrow();
+
+export const queueEntryPoint = createRequestReviewHandler(
```

Comment:
Naming: 

would `queueEntryPoint` be the only queue entrypoint of the app? Would it make sense to have a more meaningful name?